marconi-chain-index/src/Marconi/ChainIndex/Indexers.hs

```diff
@@ -593,32 +611,67 @@ runIndexers
   -> C.NetworkId
   -> ChainPoint
   -> IndexingDepth
+  -> ShouldFailIfResync
   -> Text.Text
   -> [(Worker, Maybe FilePath)]
   -> IO ()
-runIndexers socketPath networkId cliChainPoint indexingDepth traceName list = do
+runIndexers socketPath networkId cliChainPoint indexingDepth (ShouldFailIfResync shouldFailIfResync) traceName indexerList = do
```

Comment:
I suggest to create a story to introduce a config object, it would be better than this huge mess of parameters (it's out of the scope of this story, but this new parameter makes me think it's time to do so)

Reply:
Very good point. Created the story and we will merge this PR. Thanks :)